fedimint-api/src/task.rs

```diff
@@ -445,3 +445,76 @@ mod imp {
         }
     }
 }
+
+#[macro_export]
+macro_rules! async_trait {
```

Comment:
Having it named the same as underlying will confuse people, imo.

Reply:
`async_trait_maybe_send` ?

Reply:
sounds better and easier to find in the code base